src/resolvers/address-resolvers.ts

```diff
@@ -0,0 +1,42 @@
+import { AddressService } from '../service/address-service';
+import { Resolver, Mutation, Arg, Query, Ctx } from 'type-graphql';
+import { AddressModel } from '../model/address-model';
+import { CreateAddressInput } from './input-validation/address-input-validation';
+import { IMyContext } from '../types/type-context';
+import jwtUtil from '../utils/jwt-util';
+import { Inject, Service } from 'typedi';
+
+@Service()
+@Resolver()
+export class AddressResolvers {
+  private readonly AddressService: AddressService;
+
+  constructor(@Inject() AddressService: AddressService) {
+    this.AddressService = AddressService;
+  }
```

Comment:
aqui poderia passar a classe direto no constructor que o typeDI deveria pegar a instancia
```suggestion
  constructor(private readonly addressService: AddressService) {}
```